source/faq.txt

```diff
@@ -42,7 +42,8 @@ How Can I Prevent the Driver From Hanging During Connection or From Spending Too
 To prevent the driver from hanging during connection or to prevent the
 driver from spending too long trying to reach unreachable replica sets,
 you can set the ``connectTimeoutMS`` option of your
-``connection-settings``. Generally, you should ensure that the
```

Comment:
Are connection-settings and connection options two separate things? I feel like a link for "connection-settings" would make sense if they are distinct, and if they are the same we should probably standardize the name and link at the first mention instead of the second. Apologies if these are separate concepts!

Reply:
I'm not sure where "connection settings" came from. I've replaced them with "connection options" in this PR. Were there other mentions of "connection settings" that I missed?